### PR TITLE
[Security certifications] - Merge changes into upstream feature branch

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Added new security certifications bubble

## QA

- PR's have been reviewed but please check against the [brief](https://docs.google.com/document/d/1cBC6Q6yeKuYejxLEDAd1YE6mpNsfmQXp9h6hP4C7DuI/edit#) and copy docs.

### CODE REVIEW

File: templates/security/cis.html
Comment (anthonydillon) [inline templates/security/cis.html:38]: This is unusual, can we remove these br's?

File: templates/security/cis.html
Comment (anthonydillon) [inline templates/security/cis.html:60]:
```suggestion
    <table>
```

File: templates/security/cis.html
Comment (anthonydillon) [inline templates/security/cis.html:6]: Heading in link

File: templates/security/fips.html
Comment (anthonydillon) [inline templates/security/fips.html:92]: This table content should be left aligned

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Few comments inline to address before releasing. 

I am concerned about the navigation here @pmahnke are you aware this?

File: navigation.yaml
Comment (bethcollins92) [reply navigation.yaml:1477]: Thanks @anthonydillon will take a look at these when I can. I did have a call with @pmahnke and ask him about this navigation but obviously can change it if it's confusing

Comment (anthonydillon) [reply navigation.yaml:1477]: I assume this is due to the length of the navigation items under security. Maybe a new section such as "Features", "Compatibility", etc.

Comment (pmahnke) [reply navigation.yaml]: My mistake entirely, I have commited a fix...

```
security:
  title: Security
  path: /security

  children:
    - title: Overview
      path: /security
    - title: ESM
      path: /security/esm
    - title: Livepatch
      path: /security/livepatch
    - title: FIPS
      path: /security/certifications#fips
    - title: Hardening
      path: /security/certifications#cis
    - title: Certifications
      path: /security/certifications
      children:
        - title: Overview
          path: /security/certifications
        - title: FIPS
          path: /security/fips
        - title: Common Criteria
          path: /security/cc
        - title: DISA-STIG
          path: /security/disa-stig
        - title: CIS
          path: /security/cis
    - title: CVEs
      path: /security/cve
    - title: Notices
      path: /security/notices
      persist: True
    - title: OVAL
      path: /security/oval
    - title: Docker Images
      path: /security/docker-images
    - title: Contact us
      path: /security/contact-us
      hidden: True
    - title: Thank you
      path: /security/thank-you
      hidden: True
```

**Comment by anasereijo** [comment]
not sure if this is the correct PR but this table on /security/cis doesn't look right:

Screenshot 2021-06-09 at 16 56 11

the first col should be wider. something like this:

File: templates/security/certifications.html
Comment (nmav) [inline templates/security/certifications.html:171]: Common Criteria on 20.04 is not under validation, we should either put a dash or leave it empty, whatever looks better.

File: templates/security/fips.html
Comment (nmav) [inline templates/security/fips.html:61]: A typo here

File: navigation.yaml
Comment (bethcollins92) [reply navigation.yaml]: > not sure if this is the correct PR but this table on /security/cis doesn't look right:
> 
> Screenshot 2021-06-09 at 16 56 11
> 
> the first col should be wider. something like this:
> 
> Screenshot 2021-06-09 at 16 58 10

Yep, there is a comment about this in the PR. Will try and fix it asap

**Comment by bethcollins92** [comment]
@anasereijo Yeah, there is a comment already about this in the PR. Will try and look at it ASAP

File: templates/security/cis.html
Comment (nmav) [inline templates/security/cis.html:58]: Let's make it shorter:
"Which versions of Ubuntu have CIS tooling?"

I'll update the copydoc

Comment (bethcollins92) [reply templates/security/cis.html:58]: I can change it this time, but just for future reference we don't normally do copy updates on the review stage of the process. The copy should be finalised before we start building.

File: navigation.yaml
Comment (nmav) [reply navigation.yaml]: FIPS and Hardening are duplicate both in the top and the sub-certifications menu. They don't need to be in both.

Certifications menu item should be "Hardening & Certifications"

File: templates/security/cis.html
Comment (nmav) [reply templates/security/cis.html:58]: I understand but to avoid a misunderstanding in that case I specifically asked for a preview when the pages are ready because (a) these need to be approved and (b) there is no way to know how the web pages will exactly look like when making a google doc. In this particular case the sentence looked too long and out of place in the page.

File: templates/security/cc.html
Comment (bethcollins92) [reply templates/security/cc.html:7]: Sorry @anthonydillon , @petesfrench and I aren't sure what you mean by this?

Comment (anthonydillon) [reply templates/security/cc.html:7]: I mean just remove the `#heading=h.335zx3wyom0b` from the link.

**Comment by bethcollins92** [comment]
Thanks @anthonydillon - @petesfrench and I have resolved comments, ready for re-review

File: navigation.yaml
Comment (anthonydillon) [reply navigation.yaml]: Sorry again, it's strange that the Overview link goes to the top level section in the secondary nav.

File: templates/security/cc.html
Comment (anthonydillon) [inline templates/security/cc.html:14]: What's the purpose of the spans here?

File: templates/security/certifications.html
Comment (anthonydillon) [inline templates/security/certifications.html:84]: This content should not be centered.

File: templates/security/cis.html
Comment (anthonydillon) [inline templates/security/cis.html:70]: These should be centered

File: templates/security/fips.html
Comment (anthonydillon) [inline templates/security/fips.html:101]: These should be left aligned the same as the table heading

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Sorry, looking better but a few comments and the navigation still links strangely. The overview link, highlights the to breadcrumb.

File: templates/security/certifications.html
Comment (bethcollins92) [reply templates/security/certifications.html:84]: Is this better? I thought it looked more even with columns centered.  

Screenshot 2021-06-15 at 08 43 25

**Comment by bethcollins92** [comment]
Thanks @anthonydillon - I removed the overview from the secondary nav and just left certifications, not sure we need the overview when it's pointing to the same page?

File: templates/security/certifications.html
Comment (anthonydillon) [inline templates/security/certifications.html:82]: If its just a tick in the cell the content can we centralised as it is in other pages in this PR

File: templates/security/certifications.html
Comment (anthonydillon) [inline templates/security/certifications.html:102]: We have a mix of ticks here

**Review by anthonydillon — APPROVED** [review]
LGTM with two small suggestions

File: templates/security/cc.html
Comment (nmav) [inline templates/security/cc.html:74]: This section was invisible while reading the page. Invisible in the sense that I stopped reading mechanically at the previous section as this looked like the bottom footer. This table is part of the previous section, so I do not think it should have the same color as the footer and different than the section itself.

File: templates/security/certifications.html
Comment (bethcollins92) [reply templates/security/certifications.html:82]: done

File: templates/security/certifications.html
Comment (bethcollins92) [reply templates/security/certifications.html:102]: 👍🏼  I've swapped out the green tick icons for assets

File: templates/security/livepatch.html
Comment (bethcollins92) [reply templates/security/livepatch.html:234]: @petesfrench

File: templates/security/disa-stig.html
Comment (bethcollins92) [reply templates/security/disa-stig.html:49]: I'm guessing you also mean in front of 18.04 and 20.04 too?

Comment (bethcollins92) [reply templates/security/disa-stig.html:49]: also on the overview page, I'll add it there for consistency

**Comment by bethcollins92** [comment]
Thanks @nmav 👍🏼 , I've made those suggested changes if you want to have another look

**Comment by SirSamTumless** [comment]
@nmav Could you please review this and give it a PdM +1 label if you're happy with it?

**Comment by SirSamTumless** [comment]
@anthonydillon Are these good to go?

**Comment by anthonydillon** [comment]
Yes it would look like it. @bethcollins92 would need to remove the don't merge label as there was a reason for it.

**Comment by bethcollins92** [comment]
@petesfrench

**Comment by bethcollins92** [comment]
@carkod  is working on the copy updates before we merge - made another PR [here](https://github.com/canonical-web-and-design/ubuntu.com/pull/10011) (as this was from my fork to master instead of from `canonical-web-and-design:security-certifications` to master). 

I changed the base of this to canonical-web-and-design:security-certifications so it's only some small changes from Pete in the diff now. 

Carlos can then make his copy update PRs against `canonical-web-and-design:security-certifications` and when they're all reviewed we can do one final review of the [feature branch PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/10011) before we merge it all to master

**Pull request merged**
